admin/system/delete.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    $tabel = $_GET["page"];
```

Comment:
Codacy found an issue: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)